### ISSUE TITLE
Crash bug: Fatal Python error: (pygame parachute) Segmentation Fault in Ubuntu

### ISSUE THREAD

**Issue by 4015-al** [singularity/singularity#317]
**Please include the log file**

```sh
➜  ~ singularity 
Singularity 1.00 (commit: 2ebc2f3f2059b96885416167363bde2e27ece106)
Running under Python 3.8.10 (default, Sep 28 2021, 16:10:42) [GCC 9.3.0]
pygame 2.0.2 (SDL 2.0.16, Python 3.8.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
The error-log configured as /home/user/.local/share/singularity/log/error.log (lazily created when something is logged)
Fatal Python error: (pygame parachute) Segmentation Fault
Python runtime state: initialized

Current thread 0x00007fd2fb7a4700 (most recent call first):
<no Python frame>

Thread 0x00007fd306b64740 (most recent call first):
  File "/mnt/singularity-1.00/singularity/code/graphics/dialog.py", line 272 in handle
  File "/mnt/singularity-1.00/singularity/code/graphics/dialog.py", line 231 in show
  File "/mnt/singularity-1.00/singularity/code/graphics/dialog.py", line 125 in call_dialog
  File "/mnt/singularity-1.00/singularity/code/screens/map.py", line 498 in show_message
  File "/mnt/singularity-1.00/singularity/code/player.py", line 569 in remove_bases
  File "/mnt/singularity-1.00/singularity/code/player.py", line 422 in give_time
  File "/mnt/singularity-1.00/singularity/code/screens/map.py", line 636 in on_tick
  File "/mnt/singularity-1.00/singularity/code/graphics/dialog.py", line 405 in call_handlers
  File "/mnt/singularity-1.00/singularity/code/graphics/dialog.py", line 390 in handle
  File "/mnt/singularity-1.00/singularity/code/graphics/dialog.py", line 231 in show
  File "/mnt/singularity-1.00/singularity/code/safety.py", line 64 in safe_call
  File "/mnt/singularity-1.00/singularity/code/screens/map.py", line 580 in show
  File "/mnt/singularity-1.00/singularity/code/graphics/dialog.py", line 125 in call_dialog
  File "/mnt/singularity-1.00/singularity/code/screens/main_menu.py", line 104 in new_game
  File "/mnt/singularity-1.00/singularity/code/graphics/button.py", line 247 in activated
  File "/mnt/singularity-1.00/singularity/code/graphics/button.py", line 213 in activate_with_sound
  File "/mnt/singularity-1.00/singularity/code/graphics/button.py", line 199 in handle_event
  File "/mnt/singularity-1.00/singularity/code/graphics/dialog.py", line 405 in call_handlers
  File "/mnt/singularity-1.00/singularity/code/graphics/dialog.py", line 390 in handle
  File "/mnt/singularity-1.00/singularity/code/graphics/dialog.py", line 231 in show
  File "/mnt/singularity-1.00/singularity/__init__.py", line 382 in main
  File "/mnt/singularity-1.00/singularity/__main__.py", line 26 in <module>
  File "/usr/lib/python3.8/runpy.py", line 87 in _run_code
  File "/usr/lib/python3.8/runpy.py", line 194 in _run_module_as_main
```
`error.log` file is not created

**To Reproduce**
It happens frequently, after ~100 to 150 in-game days, I don't know how to reproduce it.

**Additional context**

What I did : 

1. download the `tar`  file from [singularity web site](http://www.emhsoft.com/singularity/)
2. move the file to a desired location, and uncompressed it with `tar -xf` command
3. run the game using `python3 -m singularity` (from a shell function)
4.  add the music folder and its files
5. play for a short while
6. get this crash

- Operating System:
```sh
➜  ~ lsb_release -a               
LSB Version:	core-11.1.0ubuntu2-noarch:security-11.1.0ubuntu2-noarch
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal
```

-  information about: **python**, **pip**, **pygame** and **numpy**
```sh
➜  ~ python -V  
Python 3.8.10                                                                                                                  /0.0s
➜  ~ pip -V
pip 21.3 from /home/user/.local/lib/python3.8/site-packages/pip (python 3.8)                                                    /0.2s
➜  ~ pip show pygame
Name: pygame
Version: 2.0.2
Summary: Python Game Development
Home-page: https://www.pygame.org
Author: A community project.
Author-email: pygame@pygame.org
License: LGPL
Location: /home/user/.local/lib/python3.8/site-packages
Requires: 
Required-by:                                                                                                                   /0.4s
➜  ~ pip show numpy 
Name: numpy
Version: 1.21.2
Summary: NumPy is the fundamental package for array computing with Python.
Home-page: https://www.numpy.org
Author: Travis E. Oliphant et al.
Author-email: 
License: BSD
Location: /home/user/.local/lib/python3.8/site-packages
Requires: 
Required-by: 
```

**Comment by leha-code** [comment]
The last big commits were two years ago: Pygame has changed a lot through that period. I will try running this game with Pygame versions 1.9.2 to 2.1.2 to find out which versions are compaptible. 1.9.2 probably will be, not sure about 2.0.1, but pygame 2.0 changed a lot of things so try 1.9.2

**Comment by nthykier** [comment]
Most likely related to pygame/pygame#3128. At this point, the only known work around is to downgrade your pygame version.

As for pygame versions, I would recommend 1.9.6 as the lowest (or maybe 1.9.4 if you cannot get 1.9.6 for some reason).  These versions have been supported for a long time so there is a high probability that the game still works with them.

**Comment by nthykier** [comment]
This should be fixed in pygame/2.5.2 or the latest version of pygame-ce.
